tests/foreman/ui/test_computeresource_vmware.py

```diff
@@ -562,7 +571,7 @@ def test_positive_virt_card(session, target_sat, module_location, module_org):
         assert virt_card['cluster'] == settings.vmware.cluster
         assert virt_card['memory'] == '5 GB'
         assert 'public_ip_address' in virt_card
-        assert virt_card['mac_address'] == settings.vmware.mac_address
+        assert virt_card['mac_address'] != ''
```

Comment:
I think we should assert here with actual mac address check here and we can run this test on either version as there'll be no change in virt card displayed, right?

Reply:
Updated.